tests/core/editable/keystrokes/delbackspacequirks/manual/noselection.html

```diff
@@ -0,0 +1,30 @@
+<div id="editor">
+	<p>Foo</p>
+</div>
+
+<script>
+
+	if ( bender.tools.env.mobile ) {
+		bender.ignore();
+	}
+
+	CKEDITOR.replace( 'editor', {
+		height: 300
+	} );
+</script>
+
+<button onclick="start()" >Start.</button>
+<script>
+	function start() {
```

Comment:
Ok I test it, and it's working as it should. So it simplifies entire test :) I make a new commit with such improvement.